packages/web/src/components/list/TabDataList.js

```diff
@@ -27,7 +28,9 @@ const TabDataList = (props) => {
 		<SingleDataList
 			{...props}
 			showSearch={props.showSearch}
-			render={({ data, value, handleChange }) => (
+			render={({
```

Comment:
@SavvyShah You don't need to connect this component, provide `total` value in the render method of `SingleDataList` and use it.